ui/dashboards/src/components/GridLayout/GridLayout.tsx

```diff
@@ -37,7 +37,7 @@ export function GridLayout(props: GridLayoutProps) {
     ...others
   } = props;
 
-  const [isOpen, setIsOpen] = useState(!!spec.display?.collapse?.open);
```

Comment:
From #561 it was more to make spec.display itself optional since our internal dashboards don't define it and [will all break](https://github.com/perses/perses/blob/c9f3a614cafd9ff9b91acd4a63c68d817a5bbca0/ui/dashboards/src/components/GridLayout/GridLayout.tsx#L60) (without an error message, took me a little bit to figure out why the entire dashboard wasn't rendering), but not a big deal to fix them since there aren't that many. Also, agreed with your comment about not setting a default for whole display since we'll need to support groups without titles

Reply:
Isn't it already optional? https://github.com/perses/perses/blob/main/ui/core/src/model/layout.ts#L21 Was there another error where we're not respecting the type?

Reply:
So just to clarify what I was thinking: if there's no `display`, we shouldn't render the title bar, but we should still render the grid's content. I think the problem here was that we were setting `isOpen` to `false` by default when there wasn't a `display` prop which caused the grid's content to remain "collapsed" and not render.